src/DI/ValidatorExtension.php

```diff
@@ -73,9 +73,15 @@ public function loadConfiguration()
 			->setClass('Symfony\Component\Validator\ConstraintValidatorFactoryInterface')
 			->setFactory('Kdyby\Validator\ConstraintValidatorFactory');
 
+		// allow use new SF 4.2 Contract component
+		$contextFactoryParams = ['translationDomain' => $config['translationDomain']];
+		if (interface_exists('\Symfony\Contracts\Translation\TranslatorInterface')) {
```

Comment:
```suggestion
		if (interface_exists('Symfony\Contracts\Translation\TranslatorInterface')) {
```

Reply:
Ok, I replace it by `Symfony\Component\Translation\TranslatorInterface`